themes/digital.gov/src/scss/new/guides/_guide-menu-bar.scss

```diff
@@ -78,3 +78,101 @@
     }
   }
 }
+
+.dg-accordion-mobile-menu {
+  position: sticky;
+  top: -1px;
+  z-index: z-index(100);
+
+  &__content {
+    background-color: color("gray-3");
+    padding: 0;
+  }
+
+  &__heading {
+    align-items: center;
+    background-color: color("gray-cool-5");
+    display: flex;
+    flex-direction: row;
+    height: 3.25rem;
+
+    svg {
+      margin-top: units(1);
+    }
+  }
+
+  &__image {
+    display: flex;
+    margin-right: 1.5rem;
+    &-icon {
+      flex: 1 0 0;
+      height: 2rem;
+      width: 2rem;
+    }
+  }
+
+  &__image-container {
+    align-items: flex-end;
+    display: flex;
+    flex: 10 0 0;
+    flex-direction: row;
+    justify-content: right;
+  }
+
+  &__image-glossary {
+    margin-left: 0.75rem;
+  }
+
+  &__button {
+    background-color: color("blue-warm-20v");
```

Comment:
The `usa-accordion__button` uses:

```css
background-image: url(../uswds/img/usa-icons/remove.svg), linear-gradient(transparent, transparent);
```

Have not found a solution to override this hard-coded svg graphic for the +/- icon.